modules/sr/robot/ruggeduino_devices.py

```diff
@@ -41,18 +45,19 @@ def read_value(self):
 
 class Led:
 
-    def __init__(self, webot, device_name):
+    def __init__(self, webot, device_name, limiter) -> None:
+        self._name = device_name
         self.webot_sensor = webot.getLED(device_name)
         self._webot = webot
-        self._last_change = 0
+        self._limiter = limiter
 
     def write_value(self, value):
-        now = self._webot.getTime()
-        diff = now - self._last_change
-        if diff < 0.6:
-            # Avoid creating strobing effects by allowing LEDs to change at most
-            # once per second.
+        if not self._limiter.can_chage():
```

Comment:
```suggestion
        if not self._limiter.can_change():
```

Reply:
I think https://github.com/srobo/competition-simulator/pull/188/commits/ba3f4cb728dc966332918a02e7a1541f4881c6b4 does this?